packages/jaeger-ui/index.html

```diff
@@ -34,19 +34,15 @@
         const JAEGER_VERSION = DEFAULT_VERSION;
         return JAEGER_VERSION;
       }
+
+      // Workaround some legacy NPM dependencies that assume this is always defined.
+      window.global = {};
+      // Avoid noise from redux-form until https://github.com/redux-form/redux-form/pull/4723 is released.
+      window.module = {};
     </script>
   </head>
   <body>
     <div id="jaeger-ui-root"></div>
-    <!--
-      This HTML file is a template.
-      If you open it directly in the browser, you will see an empty page.
-
-      You can add webfonts, meta tags, or analytics to this file.
-      The build step will place the bundled scripts into the <body> tag.
```

Comment:
It does. I only removed this comment because it was part of the CRA default template—but on second thought, I think I'll keep it because what it says is still completely applicable.

Reply:
would be good to link to Vite's documentation that explains what/how this is done